R/ResampleInstances.R

```diff
@@ -14,6 +14,19 @@ instantiateResampleInstance.CVDesc = function(desc, size, task = NULL) {
   makeResampleInstanceInternal(desc, size, test.inds = test.inds)
 }
 
+instantiateResampleInstance.SpCVDesc = function(desc, size, task = NULL) {
+  coords = data.frame(task$env$data$x, task$env$data$y)
+  # perform kmeans clustering
+  inds = kmeans(coords, centers = desc$iters)
+  inds = factor(inds$cluster)
+
+  # get row indices of test set from clustering
+  test.inds = lapply(levels(inds), function(x, spl)
+    which(spl == x), spl = inds)
```

Comment:
I don't understand this code. Why does it give the test indices? Doesn't this simply assign everything to their clusters?